R/ggplotly.R

```diff
@@ -806,7 +806,12 @@ gg2list <- function(p, width = NULL, height = NULL,
       
       # do some stuff that should be done once for the entire plot
       if (i == 1) {
-        axisTickText <- longest_element(axisObj$ticktext)
+        # Split ticktext elements by "\n"  to account for linebreaks
+        axisTickText <- sapply(as.character(axisObj$ticktext), strsplit, split = "\n", USE.NAMES = FALSE)
```

Comment:
Is the `sapply()` necessary here? Couldn't it just be `strsplit(axisObj$ticktext, split = "\n", USE.NAMES = FALSE)`

Reply:
Yep. You are right. `sapply` isn't necessary. I will fix that and also try to add some tests.

PS: I will keep your hint on the use `sapply` in mind.

Reply:
Removed the unnecessary `sapply `and added some tests. Visual tests have been run against the orca image.